### ISSUE TITLE
If ... Then ... Else syntax in the same line is not working with Set statement

### ISSUE THREAD

**Issue by cristianbuse** [WaynePhillipsEA/twinbasic#148]
**Describe the bug**
If ... Then ... Else syntax in the same line is not working together with the ```Set``` statement.

**To Reproduce**
```VBA
Public Sub Main()
    Dim arr(0 To 1) As Variant
    Dim v As Variant
    
    If True Then v = 123 Else v = 456               'This works
    If False Then Set arr(1) = v Else arr(1) = v    'This doesn't work

    MsgBox arr(1)
End Sub
```

**Expected behavior**
Code is valid and works in VBA. Same code should compile and work in TB.

**Screenshots**
![test](https://i.stack.imgur.com/z1yCf.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise V1909
 - twinBASIC compiler v0.9.2720

**Additional context**
None

**Comment by WaynePhillipsEA** [comment]
Thanks for reporting, we'll get this fixed.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.509, thanks for reporting
